image.yaml

```diff
@@ -46,14 +46,10 @@ modules:
             version: '1.0'
 
 artifacts:
-      - md5: 34a8787d5f5cdf54d5c1477ab1bdf47b
-        name: jboss-eap-7.2.7-patch
-        target: jboss-eap-7.2.7-patch.zip
-        url: http://$DOWNLOAD_SERVER/devel/candidates/JBEAP/JBEAP-7.2.7.CP-CR3/jboss-eap-7.2.7.CP-CR3-patch.zip
-      - md5: c8ecfa7a0c68c3c21a3333b1e5a9c343
-        name: jbeap-18811
-        target: jbeap-18811.zip
-        url: http://$DOWNLOAD_SERVER/devel/candidates/JBSSO/JBSSO-7.3.7.CR1/jbeap-18811.zip
+      - md5: 1810475433681fb32f96dfd5089b5814
+        name: jboss-eap-7.2.8.CP-CR1-patch
+        target: jboss-eap-7.2.8.CP-CR1-patch.zip
+        url: http://$DOWNLOAD_SERVER/devel/candidates/jboss/eap/JBEAP-7.2.8.CP-CR1/jboss-eap-7.2.8.CP-CR1-patch.zip
```

Comment:
I cannot check this since for the moment, the response from the http://$DOWNLOAD_SERVER/devel/candidates/jboss/eap/JBEAP-7.2.8.CP-CR1 is _Not Found_.

Reply:
Seems that the correct path would now be `/devel/candidates/middleware/eap/JBEAP-7.2.8.CP-CR1/jboss-eap-7.2.8.CP-CR1-patch.zip`

Reply:
The URL suffix (after the hostname) seems to be OK. Verified the md5 too before.

Reply:
Both of the paths seems to be WFM when downloading via wget

Reply:
Tried again: http://$DOWNLOAD_SERVER/devel/candidates/jboss/eap/JBEAP-7.2.8.CP-CR1/` leads to 404 Not found. There's something fishy going on, but the safest bet seems now the `/devel/candidates/middleware/eap/JBEAP-7.2.8.CP-CR1/` one.

Reply:
Candidates Brno mirror is down (few weeks actually). I'm not sure from what geological location  is this code executed but we have to use Boston mirror.

Reply:
@drichtarik Could you change the URL infix to the `/devel/candidates/middleware/eap/JBEAP-7.2.8.CP-CR1/` one, yet?

Reply:
Right, but this is not about the specific mirror, but about the directory path in the mirror. Both of the paths work from me (and their md5sum matches). Closing, we can address this later.